File: solutions/python/VB_T2_015_is_longer_than.py
"""VB-T2-015: Is Longer Than -- Python baseline."""
def is_longer_than(s: str, threshold: int) -> bool:
    return len(s) > threshold

if __name__ == "__main__":
    assert is_longer_than("hello", 3) is True
    assert is_longer_than("hi", 5) is False
    assert is_longer_than("", 0) is False
    assert is_longer_than("test", 4) is False
    assert is_longer_than("test", 3) is True
    print("VB-T2-015 ok")

File: solutions/typescript/VB_T2_015_is_longer_than.ts
// VB-T2-015: Is Longer Than -- TypeScript baseline
function isLongerThan(s: string, threshold: number): boolean { return s.length > threshold; }
console.assert(isLongerThan("hello", 3) === true);
console.assert(isLongerThan("hi", 5) === false);
console.assert(isLongerThan("", 0) === false);
console.assert(isLongerThan("test", 4) === false);
console.assert(isLongerThan("test", 3) === true);
